types/youtube-player/youtube-player-tests.ts

```diff
@@ -53,6 +53,7 @@ player.loadVideoById('doesNotExist');
 player.playVideo();
 player.pauseVideo();
 player.setSize(320, 200);
+console.log('player size:', player.getSize());
```

Comment:
Please do not rely on `console.log` for testing code as it does not allow for automated verification.

I suggest you try something like this:

```typescript
const playerSize: Promise<sizeType> = player.getSize();
```

Reply:
good point, adjusted.